src/coreclr/CMakeLists.txt

```diff
@@ -129,6 +129,8 @@ endif(CLR_CROSS_COMPONENTS_BUILD)
 #-------------------
 include(pgosupport.cmake)
 
+add_subdirectory(src/librariesshims)
```

Comment:
Nit: `librariesshims` is hard to parse name and associate with `libraries/native`. What about `libraries-native` ?

Reply:
Good idea, will update.